src/plays/sticky-notes/styles.css

```diff
@@ -0,0 +1,6 @@
+/* enter stlyes here */
+body {

```

Comment:
Do not override the selectors like this. Please add a unique class name as prefix which is your play specific.